trace_decoder/src/lib.rs

```diff
@@ -104,10 +104,16 @@ use evm_arithmetization::GenerationInputs;
 use keccak_hash::keccak as hash;
 use keccak_hash::H256;
 use mpt_trie::partial_trie::{HashedPartialTrie, OnOrphanedHashNode};
+use plonky2::field::goldilocks_field::GoldilocksField;
+use plonky2::hash::hash_types::NUM_HASH_OUT_ELTS;
 use processed_block_trace::ProcessedTxnInfo;
 use serde::{Deserialize, Serialize};
 use typed_mpt::{StateTrie, StorageTrie, TrieKey};
 
+/// The base field on which statements are being proven.
+// TODO(Robin): https://github.com/0xPolygonZero/zk_evm/issues/531
+pub type Field = GoldilocksField;
```

Comment:
very nit/style:

I think this could be confusing because it's name collides with a type parameter name:
```rust
type Inner = u8;
struct Foo<Inner>(Inner);
```
When I see `Foo<Inner>`, is it like `Vec<u8>` or `Vec<T>`.

I prefer to use `T` as a suffix for actually descriptive type parameters.
```rust
type Field = GoldilocksField.
struct Foo<FieldT>(FieldT);
```